app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if current_user %>
+  <h1><%= "Welcome, #{current_user.first_name || current_user.email}." %></h1>
+  <%= link_to 'Sign Out', logout_path, method: :delete %>
+
+  <h2>My Groups</h2>
+  <% if @user.groups.nil? %>
```

Comment:
unsure if this will work -- I don't think an empty collection is nil

Reply:
This file is unused and its functionality has been moved to the `dashboard#show` action. The file will be removed. There is a similar conditional check within `dashboard/show.html.erb` which I will change.